app/views/site/render_gene_expression_plots.js.erb

```diff
@@ -81,6 +85,8 @@ console.log('render status of box plot at end: ' + $('#expression-plots').data('
 console.log('render status of scatter plot at end: ' + $('#expression-plots').data('scatter-rendered'));
 console.log('render status of reference plot at end: ' + $('#expression-plots').data('reference-rendered'));
 
```

Comment:
the changes in this file are nice -- they show that you encapsulated the ideogram logic cleanly.